content/800-data-platform/100-accelerate/200-getting-started.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Getting started'
+metaTitle: 'Getting started with Accelerate'
+metaDescription: 'Learn how to get up and running with Accelerate.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+</TopBlock>
+
+## Prerequisites
+
+To be able to participate in Accelerate’s Early Access program, you need the following prerequisites:
+
+- A GitHub account.
+- A project that uses [Prisma Client](/docs/concepts/components/prisma-client) `4.16.1` or higher.
```

Comment:
@luanvdw you mentioned this morning that interactive transactions would only be available in `5.0.0` with Accelerate. Is that accurate? I couldn't find anything about it in the [release notes](https://github.com/prisma/prisma/releases) (except for this [issue](https://github.com/prisma/prisma/issues/17948) that has been closed in [`4.16.0`](https://github.com/prisma/prisma/releases/tag/4.16.0)).

Can you confirm that `5.0.0` is indeed the versoin that supports interactive transactions? If so, we'll update this bullet with a note in parantheses that if people are using interactive transactions that they need to use `5.0.0`.

Reply:
@nikolasburk thanks for checking 🙏 iTX is supported in `5.1.1`. It will still be broken in `5.0.0`